src/config.js

```diff
@@ -226,6 +229,13 @@ export default class Config {
       constants.NETWORK_CONCURRENCY
     );
 
+    this.childConcurrency = (
+      opts.childConcurrency ||
+      Number(this.getOption('child-concurrency')) ||
+      Number(process.env.CHILD_CONCURRENCY) ||
```

Comment:
Do we need an extra environment variable for this?
Would not CLI arg and .yarnrc setting be enough?